src/wp-content/themes/twentynineteen/sass/blocks/_blocks.scss

```diff
@@ -930,18 +934,22 @@
 
 	//! Font Sizes
 	.has-small-font-size {
+		--wp--preset--font-size--small: 0.88889em;
```

Comment:
The current setup doesn't allow for using `--wp--preset--font-size--small: $font__size-sm`. I've tried to upgrade to a newer setup but ran into a few issues and couldn't. This is the raw value after compiling SCSS into CSS.